dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/EnrollmentService.java

```diff
@@ -142,14 +125,6 @@ public interface EnrollmentService {
    */
   List<Enrollment> getEnrollments(EnrollmentQueryParams params);
```

Comment:
Matbe not for this PR but this seems to be used only in tests and it should allow to remove `EnrollmentQueryParams` altogether

Reply:
I thought finding unused code should be easy but it's not 😅 I haven't found a way to find unused code or code only used in tests on a larger scale in Intellij. I might have to go service by service and method by method 😬 

I will follow up with another PR.